ament_cmake_vendor_package/cmake/templates/vendor_package_cmake_prefix.cmake.in

```diff
@@ -0,0 +1,3 @@
+# generated from ament_cmake_vendor_package/cmake/templates/vendor_package_cmake_prefix.cmake.in
+
+list(PREPEND CMAKE_PREFIX_PATH "${CMAKE_CURRENT_LIST_DIR}/../../../opt/@PROJECT_NAME@")
```

Comment:
I'd recommend adding a comment about what the `..` are for. Is it backing out of  `share/<vendor_pkg_name>/cmake/`?

Reply:
Done: 2e114b6